Problem2.scala

```diff
@@ -0,0 +1,53 @@
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions._
+
+
+
+object Problem2 extends App{
+  
+  val sparkConf = new SparkConf()
+  sparkConf.set("spark.app.Name","my app")
+  sparkConf.set("spark.master","local[2]")
+  
+  val spark = SparkSession.builder()
+  .config(sparkConf)
+  .getOrCreate()
+  
+  val ratingsRdd = spark.sparkContext.textFile("/Users/smili/Documents/Big data -Naveena/Week 12/dataset for assignment/ratings.dat")
+    
+    val ratingMap = ratingsRdd.map(x => {
+      val fields = x.split("::")
+      (fields(0),fields(1),fields(2),fields(3))
+    })
+    
+    import spark.implicits._
+    
+    val ratingsDf=spark.createDataFrame(ratingMap)
+    .toDF("movieid","custid","rating","phoneno")
+    
+    
+      val moviesRdd = spark.sparkContext.textFile("/Users/smili/Documents/Big data -Naveena/Week 12/dataset for assignment/movies.dat")
```

Comment:
file name as input file path

`val moviesRdd = spark.sparkContext.textFile("/Users/smili/Documents/Big data -Naveena/Week 12/dataset for assignment/movies.dat")`
also check alignment